File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/ClsCallbackWeight.java
package com.example.sarahn.waterreminderapp.classes;

import android.widget.NumberPicker;

/**
 * Created by SarahN on 6/13/2017.
 */
public class ClsCallbackWeight implements NumberPicker.OnValueChangeListener {

    @Override
    public void onValueChange(NumberPicker picker, int oldVal, int newVal) {

        if(ClsBottomSheetWeight.values.length == 0){
            return;
        }else{
            ClsBottomSheetWeight.tv.setText(ClsBottomSheetWeight.values[newVal-1]);
        }
    }
}

File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/ClsCallbackWeight.kt
package com.example.sarahn.waterreminderapp.classes

import android.widget.NumberPicker
import android.widget.NumberPicker.OnValueChangeListener

/**
 * Created by SarahN on 6/13/2017.
 */
class ClsCallbackWeight : OnValueChangeListener {
    override fun onValueChange(picker: NumberPicker, oldVal: Int, newVal: Int) {
        if (ClsBottomSheetWeight.values?.size == 0) {
            return
        } else {
            ClsBottomSheetWeight.tv!!.text = ClsBottomSheetWeight.values?.get(newVal - 1) ?: ""
        }
    }
}